.github/workflows/cd.yml

```diff
@@ -56,4 +56,5 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           PYPI_TOKEN: ${{ secrets.TWINE_PASSWORD }}
+          PYPI_USERNAME: ${{ secrets.TWINE_USERNAME }}
```

Comment:
can you keep the original names? TWINE_PASSWORD and TWINE_USERNAME?

Reply:
no, this is the required name for semantic release: https://github.com/abichinger/semantic-release-pypi#environment-variables